lib/reek/cli/command/commandable.rb

```diff
@@ -0,0 +1,35 @@
+module Reek
+  module CLI
+    module Command
+      #
+      # Base class for all commands
+      #
+      # :reek:ModuleInitialize
+      module Commandable
+        private
+
+        attr_reader :options
+
+        def initialize(options)
+          @options = options
+        end
+
+        def scan_for_smells(app)
+          sources.each do |source|
+            reporter.add_examiner Examiner.new(source,
```

Comment:
I'll change this anyway after our discussion, but out of interest, why do you find this confusing?
This is something you will encounter with the majority of all mixins, won't you?